### PR TITLE
feat(cockpit): handle organization and environment commands

### LINKED ISSUE #4287: [management] move organization & environment creation to command handler

### PR DESCRIPTION
**Pull request by jhaeyaert**
Closes gravitee-io/issues#4287

### CODE REVIEW

File: src/main/resources/liquibase/changelogs/v3_4_0/schema.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+  - changeSet:
+      id: 3.4.0
```
Comment (ytvnr) [inline src/main/resources/liquibase/changelogs/v3_4_0/schema.yml]: should not it be in 3.5.0 ? (and same for the folder)

File: src/main/resources/liquibase/master.yml [hunk 2]
```diff
@@ -89,3 +89,5 @@ databaseChangeLog:
       - file: liquibase/changelogs/v3_2_0/schema.yml
   - include:
       - file: liquibase/changelogs/v3_3_0/schema.yml
+  - include:
+      - file: liquibase/changelogs/v3_4_0/schema.yml
```
Comment (ytvnr) [inline src/main/resources/liquibase/master.yml:93]: 3_5_0 ?

**Review by ytvnr — COMMENTED** [review]
comments on version

File: src/main/resources/liquibase/master.yml (hunk 2 above)
Comment (jhaeyaert) [reply src/main/resources/liquibase/master.yml:93]: You're right, this isn't the right version now (it used to be but not anymore).
Feature is now expected in 3.6.0 so I'll move to that version instead.

**Review by phiz71 — APPROVED** [review]

**Pull request merged**